### PR TITLE
Josue nov19

### PR DESCRIPTION
**Pull request by JosueValladares**

### CODE REVIEW

**Review by isharoyan — CHANGES_REQUESTED** [review]
move your comments from Feature file to Step definition class and change regression tag to @Regression (case sensitive)

**Review by isharoyan — CHANGES_REQUESTED** [review]
see previous comments

**Review by isharoyan — APPROVED** [review]

File: target/test-classes/Features/MaintenanceTab/MaintenanceFeature.feature [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        And I select a request in the "New Request" column
+        And I move request to "In Progress" column
+        And I move request to "Repaired" column
+        And I move request to "Scrap" column
+        Then I should be able to see selected request "Scrap" column
+
+    @Moving-request-backwards-in-board-status
+    Scenario: Moving Maintenance request in board status
+        # When I click on Maintenance requests option
+      And I select a request in the "Scrap" column
```
Comment (isharoyan) [inline target/test-classes/Features/MaintenanceTab/MaintenanceFeature.feature:20]: did you duplicated two scenarios? and do you really need to parametrize the steps?

File: src/test/java/com/Briteerp/Runner/Runner1.java [hunk 2]
```diff
@@ -13,7 +13,7 @@
             features = "src/test/resources/Features",
             glue = "com/Briteerp/StepDefinitions",
             dryRun = false,
-            //tags = "",
+            tags = " @Moving-request-forward-in-board-status",
```
Comment (adamrochwick) [inline src/test/java/com/Briteerp/Runner/Runner1.java:16]: remember to clear out your tags before you push! :)

File: src/test/java/com/Briteerp/StepDefinitions/Maintenance/Maintenance_POM.java [hunk 3]
```diff
@@ -49,44 +50,30 @@ public void iMoveRequestToScrapColumn() {
...
-    /*
-        @Given("user click in the Maintenance tab")
-        public void user_click_in_the_maintenance_tab() {
-
-        }
-
-
-
-
-
```
Comment (adamrochwick) [inline src/test/java/com/Briteerp/StepDefinitions/Maintenance/Maintenance_POM.java]: Loto space my guy

**Review by adamrochwick — APPROVED** [review]
Looks solid!

**Review by isharoyan — APPROVED** [review]

**Comment by JosueValladares** [comment]
Thank you guys for sharing your knowledge!!

**Pull request merged**